.eslintrc.js

```diff
@@ -20,12 +20,13 @@ module.exports = {
   rules: {
     'react/react-in-jsx-scope': 'off',
     'react/no-children-prop': 'off',
-    "unused-imports/no-unused-imports": "warn",
-    "@typescript-eslint/no-unused-vars": "off",
-    "@typescript-eslint/no-unused-imports": "off",
-    "@typescript-eslint/no-unused-expressions": "off",
-    "@typescript-eslint/no-explicit-any": "off",
-    "@typescript-eslint/no-shadow": "off",
-    "@typescript-eslint/no-use-before-define": "off",
```

Comment:
Same for this. Did you format with `yarn prettier:write` ?

Reply:
Auto fixed. If you notice you use single and double quotation marks. VSCode prettier did this.